app/models/grda_warehouse/external_referral.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse
+  # A request for a service for a household. The service is not necessarily housing.
+  class ExternalReferral < GrdaWarehouseBase
+    has_many :external_referral_clients, class_name: 'GrdaWarehouse::ExternalReferralClient', dependent: :destroy
+    has_many :external_referral_postings, class_name: 'GrdaWarehouse::ExternalReferralPosting', dependent: :destroy
+  end
+end
```

Comment:
@ttoomey these should all live underneath `/drivers/hmis/app/models` (unless @gigxz had a reason for them to be out here.)

Reply:
@gigxz, were you thinking these should be in their own driver maybe?

Reply:
Thanks! My mistake, I'll move them to the hmis namespace

Reply:
Feel free to hold off on the move until we hear from @gigxz next week, I'm not sure if we should compartmentalize this in its own driver or not.

Reply:
Good question. I was thinking all the External* models would go in `drivers/hmis_external_apis/app/models` which @yakloinsteak is creating here https://github.com/greenriver/hmis-warehouse/pull/2933. The UnitType model would go in `drivers/hmis/app/models`. @eanders what do you think of that? I think I'd prefer the HMIS driver to contain only the "core" models, and the external driver can contain as much as possible of the external business logic (some of which may be on the models).

@ttoomey for now, you can just move these to `/drivers/hmis/app/models`, and I can move them over to the external driver once it's set up (unless Elliot objects). thanks!

Reply:
That sounds like a plan to me.  Core HMIS (HUD compliant bits and other things that will be enabled in all installations in `drivers/hmis` HMIS specific API infrastructure in `drivers/hmis_external_apis`.

Reply:
@gigxz Thanks! I'll just move external_* models hmis_external_apis right now and steal the relevant bits from #2933. I'd prefer to move them now before I build on them. Sound okay?

Reply:
Sounds great, thanks

Reply:
@gigxz @eanders Any objection to using model names like this: `HmisExternalApis::Referral`, `HmisExternalApis::ReferralRequest`, etc ?

And respective table names  would be `hmis_external_referrals`, `hmis_external_referral_requests`. Or is it convention that the table prefix match the driver namespace (so `hmis_external_api_referrals`)?

Reply:
Those names are good.  No need to include the driver namespace in the db at this time.